doc/content/verification/ver-1a.md

```diff
@@ -2,47 +2,146 @@
+3. Not a typo per say, but a potential source of confusion for users is that in [!cite](ambrosek2008verification), $x=0$ represents the surface not exposed to the enclosure (where $c=0$), and $x=l$ represents the surface exposed to the enclosure. In the TMAP8 documentation, we have kept this convention to correspond to the TMAP7 case, but note that the TMAP8 input file fixes $x=0$ at the enclosure surface and $x=l$ for the outer surface.
+
+### Results
+
+[ver-1a_comparison_analytical_TMAP7_release_fraction] shows the comparison of the TMAP8 calculation and the analytical solution for release fraction provided in [!cite](ambrosek2008verification). There is agreement between the two plots.
+
+!media figures/ver-1a_comparison_analytical_TMAP7_release_fraction.png
+    style=width:50%;margin-bottom:2%
+    id=ver-1a_comparison_analytical_TMAP7_release_fraction
+    caption=Comparison of TMAP8 calculation with the analytical solution for the release fraction from [!cite](ambrosek2008verification).
+
+## Verification of the tritium flux at the outer surface of the SiC slab (TMAP7)
+
+### Analytical solution
+
+In [!cite](ambrosek2008verification), i.e., TMAP7, the verification test also tests the model's accuracy in determining the tritium flux across the outer surface of the SiC slab. Using the expression of the concentration provided above, the flux on the outer surface of the SiC slab can be derived as
+
+\begin{equation}
+    J(t) = D \frac{\partial C(x,t)}{\partial x}\Bigr|_{\substack{x=0}} = 2 S P_0 L' D \sum_{n=1}^{\infty} \frac{\exp \left(-\alpha_{n}^2 D t\right) \alpha_{n}}{(l(\alpha_{n}^2 + L'^2)+L') \sin (\alpha_{n} l)},
+\end{equation}
+which can be compared to TMAP8 predictions.
+
+!alert note  title=Typos in [!cite](ambrosek2008verification)
+Again, be aware of the typos in [!cite](ambrosek2008verification) and the different coordinates used in TMAP8 and TMAP7 discussed above.
 
-[ver-1a_comparison] shows the comparison of the TMAP8 calculation and the analytical solution. There is good agreement between the two plots.
+### Results
 
+[ver-1a_comparison_analytical_TMAP7_flux] shows the comparison of the TMAP8 calculation and the analytical solution for flux at the outer surface of the SiC slab in [!cite](ambrosek2008verification). There is agreement between the two plots.
 
-!media figures/ver-1a_comparison.png
+!media figures/ver-1a_comparison_analytical_TMAP7_flux.png
     style=width:50%;margin-bottom:2%
-    id=ver-1a_comparison
-    caption=Comparison of TMAP8 calculation with the analytical solution
+    id=ver-1a_comparison_analytical_TMAP7_flux
+    caption=Comparison of TMAP8 calculation with the analytical solution for the release fraction from [!cite](ambrosek2008verification).
+
+## Comment on verification input file versus test file
+
+!alert note  title=Verification input file versus test file
```

Comment:
I'm not sure I see exactly what you have in mind Are you satisfied with the content of the note, though? Should I just make it the same but remove the note format? 
Do you want to meet to discuss or take a stab at it?

Reply:
I edited the suggested code change to reflect exactly what I mean. I intended for it to be "at minimum let's do this change, but discuss further", but now GitHub won't let it be committed since it's now "outdated". 

The content of the note is great. I kind of envision this section heading to be in every V&V test, with a bulleted list of any differences (or a base comment that no changes are required for the test or reproducible case, similar to our SQA document comments for empty sections). So rather than something that stands out as an alert, it just needs to be expected as a matter of course for the reader to see. Having a heading followed by the alert box looked visually cluttered to me - usually I use these alert boxes to illuminate a point in the surrounding text, but this was just the text itself with double "look at me" elements (the heading at Level 2 / `##` and the alert box). I say pick one mode of display and move forward with that. My personal preference is then the heading over the alert box. What do you think?

Reply:
Thank you for the clarification, we are on the same page about the importance of this section.

I understand that the title AND the alert box is a bit much. Let's keep the title and remove the alert box, I'm fine either way.